/**:
  ros__parameters:
    output_topic: "/fv/d415/detection_fusion/rois"
    hold_frames: 10
    frame_id: ""
    diagnostic_interval_ms: 100.0
    # 画像オーバーレイ用
    input_image_topic: "/fv/selected/d415/color/image_raw"
    debug:
      enable_overlay: true
      overlay_topic: "overlay"
    # 融合パラメータ（必要なら調整）
    fusion:
      iou_threshold: 0.30
      prefer_bbox: "yolov10"
      weights:
        w_object: 0.6
        w_instance: 0.4
        bonus_overlap: 0.05
    sources:
      object:
        label: "object"
        topic: "/fv/d415/object_detection/detections"
        qos_reliability: "best_effort"
        confidence_multiplier: 1.0
        min_confidence: 0.2
        qos_depth: 5
      instance:
        label: "instance"
        topic: "/fv/d415/instance_seg/detections"
        qos_reliability: "best_effort"
        confidence_multiplier: 1.0
        min_confidence: 0.1
        qos_depth: 5
